ddtrace/encoding.py

```diff
@@ -46,14 +46,6 @@ def encode_traces(self, traces):
         normalized_traces = [[span.to_dict() for span in trace] for trace in traces]
         return self._encode(normalized_traces)
 
-    def encode_services(self, services):
```

Comment:
deprecation/no-op?

It is always so hard to tell what should be considered "internal", maybe in `1.0.0` we should have an `ddtrace.internal` module for scoping this stuff.

Reply:
Hm yeah. This should be no-op'd and deprecated instead.

The internal idea is interesting. It would definitely be nice to internalize or privatize the inner workings of the tracer so we don't have to worry about these things. I like the idea of an internal module. It's very explicit and well defined.

Reply:
Since it's pretty harmless, I opted to no no-op the method and just add a deprecation. Slightly more graceful.